vars/externalPipelineExecute.groovy

```diff
@@ -0,0 +1,42 @@
+import com.sap.piper.Utils
+
+/**
+ * externalPipelineExecute
```

Comment:
Just some thoughts on the naming, you can consider:
* Do we need the `external` part here in the first place? If I have a method named `pipelineExecute` with an URI parameter, isn't it obvious enough? 
* wrt `Pipeline`, which is a rather abstract term: What reveals the purpose of theis method more clear: `Pipeline` or  `Jenkinsfile`? I myself am not so sure about this.

Reply:
changed to ```pipelineExecute```.